internal/clients/api_client.go

```diff
@@ -431,30 +435,22 @@ func buildConnectorsClient(cfg config.Client) (*connectors.Client, error) {
 		authInterceptor = connectors.WithRequestEditorFn(basicAuthProvider.Intercept)
 	}
 
-	httpClient := &http.Client{}
```

Comment:
Instead of all the logging related changes I could also just create a new http client, with a debug transport which wraps `httpClient.Transport` which would keeps things more consistent with the current release. 

I'm not a huge fan of that since I feel like it leaves the door open to similar bugs in the future, where we actually do just want to use the http client which was built for the Kibana client.